build.sbt

```diff
@@ -6,9 +6,9 @@ import sbt.Keys._
 import sbt.Tests.{Group, SubProcess}
 import sbt.{Resolver, Test, inConfig, _}
 import uk.gov.hmrc.DefaultBuildSettings.{addTestReportOption, targetJvm}
-import uk.gov.hmrc.PublishingSettings._
 import uk.gov.hmrc.gitstamp.GitStampPlugin._
 import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin._
```

Comment:
Do we need this import (import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin._) ? It's grey on my IDEA.